tests/v4l2decode.cpp

```diff
@@ -848,6 +852,7 @@ int main(int argc, char** argv)
             }
         }
     } else if (IS_DMA_BUF() || IS_DRM_NAME()) {
+#if __ENABLE_TESTS_GLES__
```

Comment:
same thing here, the else if has to be placed within the __ENABLE_TEST_GLES__ compilation flag.